functions/exercises.py

```diff
@@ -313,7 +417,10 @@ def reorder_hyphen_sequence(string):
     :return:
     """
     # INSERT YOUR CODE HERE
-    #
+    text=string.split("-")
+    liste=sorted(text)
+    textout = "-".join(liste)
+    return textout

```

Comment:
In this case, it is a matter of opinion if you should do it in one line or as you did it.
But at least I would give better naming to your variables otherwise it can be difficult to understand what you are doing here
```suggestion
    return "-".join(sorted(string.split("-")))

```